internal/cmd/kafka/command_acl_create.go

```diff
@@ -21,10 +21,10 @@ func (c *aclCommand) newCreateCommand() *cobra.Command {
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: "You can specify only one of the following flags per command invocation: `--cluster-scope`, `--consumer-group`, `--topic`, or `--transactional-id`. For example, for a consumer to read a topic, you need to grant \"READ\" and \"DESCRIBE\" both on the `--consumer-group` and the `--topic` resources, issuing two separate commands:",
-				Code: "confluent kafka acl create --allow --service-account sa-55555 --operations READ,DESCRIBE --consumer-group java_example_group_1",
+				Code: "confluent kafka acl create --allow --service-account sa-55555 --operation READ --operation DESCRIBE --consumer-group java_example_group_1",
```

Comment:
The flag is now `operations` for `confluent kafka acl create` as of v3,  this is one of our breaking changes.

Reply:
Hi @sgagniere Thanks for taking a look. I'm assuming both Support and the Customer were using an old version (looks like they tested it yesterday). They both tested with ``--operations``and it didn't work. See ticket here: https://confluent.zendesk.com/agent/tickets/141307. I'll check with Support rep to see what version he's using.

Reply:
That's also what I figured might be happening!